zetaclient/bitcoin_client.go

```diff
@@ -67,12 +69,14 @@ type BitcoinChainClient struct {
 	stop   chan struct{}
 	logger BTCLog
 	ts     *TelemetryServer
+
+	BlockCache *lru.Cache
 }
 
 const (
 	minConfirmations = 0
 	maxHeightDiff    = 10000
-	dustOffset       = 2000
+	btcBlocksPerDay  = 144
```

Comment:
Is it the same value for testnet?

Reply:
I think there is no defined rule on `dustOffset` for neither of them (testnet, mainnet). The block time is 10 mins on average in mainnet and very variable in testnet3 (1sec to 30 mins). Looking back to this variable, the LRU cache probably can have any reasonable length (e.g. 10, 100, 500, 1000) to avoid unnecessary RPC queries as long as it won't take too much resources and impact performance. I just put a meaningful name `btcBlocksPerDay` be a bit clear for people.

Reply:
So having `btcBlocksPerDay  = 144` would not be an issue when using ZetaClient with Bitcoin testnet?